versioning/revisions.rst

```diff
@@ -76,6 +76,24 @@ Take into account that it changes the default Conan behavior. e.g:
     - If you generate and upload N binary packages for a recipe with a given revision, then if you modify the recipe, and thus the recipe
       revision, you need to build and upload N new binaries matching that new recipe revision.
 
+GIT and Line Endings on Windows
+-------------------------------
+
+.. warning::
+
+  **Problem**
+
+  Git will (by default) checkout files in Windows systems using CRLF line endings, effectively producing different files. As files are different, the Conan revisions will be different from the revisions computed in other platforms such as Linux, resulting in missing the respective binaries in the other revision. 
```

Comment:
1. I don't think it's unique to Git. e.g. subversion as exactly the same problem (`svn:eol-style`)
2. the problem is more broad, it's not just CRLF vs LF, but generic SCM config difference. there are other config settings which may lead into the exactly same problem (e.g. `core.symlinks` is another configuration causing similar problems)